backend-cf/.env.example

```
@@ -0,0 +1,37 @@
+# Sokuji Backend Environment Variables
+# Copy this file to .env and fill in your values
+
+# Environment
+ENVIRONMENT=production
+
+# Clerk Authentication (Required)
+# Get these from https://dashboard.clerk.com
+CLERK_SECRET_KEY=sk_live_your_clerk_secret_key
+CLERK_PUBLISHABLE_KEY=pk_live_your_clerk_publishable_key
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLERK_PUBLISHABLE_KEY key should go before the CLERK_SECRET_KEY key

(UnorderedKey)